cpp/daal/src/algorithms/k_nearest_neighbors/oneapi/bf_knn_classification_predict_kernel_ucapi_impl.i

```diff
@@ -101,11 +108,27 @@ services::Status KNNClassificationPredictKernelUCAPI<algorithmFpType>::compute(c
     const size_t nTrainFeaturesSizeT = points->getNumberOfColumns();
 
     DAAL_CHECK(nQueryRowsSizeT <= maxInt32AsSizeT, services::ErrorIncorrectNumberOfRowsInInputNumericTable);
-    DAAL_CHECK(nLabelRowsSizeT <= maxInt32AsSizeT, services::ErrorIncorrectNumberOfRowsInInputNumericTable);
```

Comment:
Should be checked if labels are presented

Reply:
I'll fix it and all sdl compliance problems

Reply:
Fixed. Please check